src/menu.tsx

```diff
@@ -153,13 +153,15 @@ const Menu: React.FC<MenuProps> = ({renderTarget, renderMenu, width, position =
             style={{
                 ...assignInlineVars({
                     [styles.vars.width]: width ? `${width}px` : '100%',
-                    [styles.vars.top]: itemsComputedProps.top ?? '',
```

Comment:
in SSR, the `''` is removed from `style`, but not in client side.
I haven't been able to reproduce this with our ssr tests, because we are using `renderToString` api, that seems to respect the original `''`. I managed to replicate the issue in next.js using the beta `app/` directory, but next.js is probably using `renderToReadableStream` or `renderToPipeableStream` instead of `renderToString` in SSR

Reply:
an alternative could be to set the style to "initial", "none", 0..